woocommerce/woocommerce
# Update illustrations and copy of task list experiment 1

### All Submissions:






### Changes proposed in this Pull Request:

This PR add's several new illustrations to use for individual tasks and updates the copy. There was also some minor css changes to match the wireframe more accurately.

Closes #32802 

### How to test the changes in this Pull Request:

1. Checkout branch on fresh site & install the WooCommerce Admin Test Helper
2. Enable usage tracking
3. Complete OBW without any of the free features
4. Visit Tools > WCA Test Helper > Experiments and enable `woocommerce_tasklist_setup_experiment_1_2022_04`
5. Open up the wireframe in a separate window G9vi4e286yjdzyIdg8hBb1-fi-735%3A83647 under **Treatment 1**
6. Now with the wireframe side by side, complete each task in the task list and match the copy and illustration of each task (you can skip the uncompleted store details version, there is a separate issue for that).
7. When you completed the task list fully, click the Store details task and step through the onboarding installing **WooCommerce Payments** as part of the free features this time.
8. Go back to the Homescreen and check if the **WooCommerce Payments** task matches that of the wireframe (the header that is, as we have another issue for the ToS within the actual task as displayed in the wireframe - https://github.com/woocommerce/woocommerce/issues/32762 ).
9. Visit Tools > WCA Test Helper > Experiments and enable `woocommerce_tasklist_setup_experiment_2_2022_04` and disabling the other experiment.
10. Go back to the home screen and check if the headers of each panel still look correct.

### Other information:




### FOR PR REVIEWER ONLY:
